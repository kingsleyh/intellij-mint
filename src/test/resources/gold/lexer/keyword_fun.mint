module A {

  fun go() : Void {
    void
  }

  fun goo : Void {
    void
  }

  fun go(a : String, b : Number) : Void {
    void
  }

}