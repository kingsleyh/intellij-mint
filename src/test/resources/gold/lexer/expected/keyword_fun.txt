module ('module')
WHITE_SPACE (' ')
IDENT ('A')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
IDENT ('go')
( ('(')
) (')')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Void')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
void ('void')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
IDENT ('goo')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Void')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
void ('void')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n  ')
fun ('fun')
WHITE_SPACE (' ')
IDENT ('go')
( ('(')
IDENT ('a')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('String')
, (',')
WHITE_SPACE (' ')
IDENT ('b')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Number')
) (')')
WHITE_SPACE (' ')
: (':')
WHITE_SPACE (' ')
IDENT ('Void')
WHITE_SPACE (' ')
{ ('{')
WHITE_SPACE ('\n    ')
void ('void')
WHITE_SPACE ('\n  ')
} ('}')
WHITE_SPACE ('\n\n')
} ('}')